src/forms/ThresholdsSelection.tsx

```diff
@@ -92,120 +91,134 @@ class ThresholdsSelectionInput extends Component<MyProps & InjectedIntlProps, My
-                                }
+                                className={comparison === "<" ? (
+                                    `${styles.SelectedButton}`
+                                ) : (
+                                    `${styles.SelectedButton} ${styles.UnselectedButton}`
+                                )}
                                 onClick={() => valueChanged({
-                                        comparison: "<",
-                                        thresholds: thresholds
-                                    })
-                                }
+                                    comparison: "<",
+                                    thresholds: thresholds
+                                })}
                             >
                                 <FormattedMessage
                                     id="notifications_app.lower_than"
                                     defaultMessage="lower than"
                                 />
                                 &nbsp;&lt;
                             </button>
-                            <label htmlFor="warning_label">
-                                <FormattedMessage
-                                    id="notifications_app.threshold_name"
-                                    defaultMessage="Name"
-                                />
-                            </label>
-                            <div className={styles.ThresholdNames}>
-                                {thresholds.map((threshold, i) => (
-                                    <div key={i}>
-                                        <span>{threshold.warning_level}</span>
-                                        <span
-                                            className={styles.ThresholdDelete}
-                                            onClick={() => this.handleRemoveThreshold(i)}
-                                        >
-                                            &#10007;
-                                      </span>
-                                    </div>
-                                ))}
-                            </div>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td className={styles.ThresholdLabel}>
+                            <FormattedMessage
+                                id="notifications_app.threshold_value"
+                                defaultMessage="Value"
+                            />
+                        </td>
+                        <td className={styles.ThresholdLabel}>
+                            <FormattedMessage
+                                id="notifications_app.threshold_name"
+                                defaultMessage="Name"
+                            />
+                        </td>
+                    </tr>
+                    {thresholds.sort((a, b) =>
```

Comment:
I like it how you combined two seperate loops and intoo this single loop.
Defenitely an improvement !
The previous code was completely wrong here and you noticed it correctly.